infra/lambda/datasets-endpoint/test_function.py

```diff
@@ -0,0 +1,307 @@
+"""
+Basic Lambda function tests. Working Data Lake AWS environment is required
+(run '$ cdk deploy' before running tests).
+"""
+
+import logging
+import re
+import uuid
+
+import function  # pylint:disable=import-error
+import pytest
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def test_request_no_method():
+    """Try request not containing method."""
+
+    body = {}
+
+    resp = function.lambda_handler({"body": body}, "context")
+    logger.info(resp)
```

Comment:
Logs level was changed to debug in bd849f270fe1fce9610daa9aedbb3e78feb32466 , but it would be nice to automatically print response if test fails as you suggest. I'll investigate more how to do it.